dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -433,9 +432,9 @@ public void Test_Create_App_Descriptor_Then_Create_App_Integration_Then_Delete_A
 
                 //Secrets are destroyed for security every time. Making the form useless. They need to be re-generated every time.
                 final Map<String, Input> inputParamMap = ImmutableMap.of(
-                        "param1", Input.newInputParam("val-1".toCharArray(),false),
-                        "param2", Input.newInputParam("val-2".toCharArray(),false),
-                        "param3", Input.newInputParam("val-3".toCharArray(),false));
+                        "param1", Input.newInputParam("val-1".toCharArray()),
+                        "param2", Input.newInputParam("val-2".toCharArray()),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "val-2" appears 4 times in this file; the first occurrence is on line 415](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)